### ISSUE TITLE
fibo-quickstart.ttl missing 97 prefixes

### ISSUE THREAD

**Issue by VladimirAlexiev** [edmcouncil/fibo#957]
I assume dev.fibo-quickstart.ttl should include all ontologies.
The following 97 ontologies are loaded by LoadFIBODev.ttl but have no prefixes defined in fibo-quickstart.ttl:

```
https://spec.edmcouncil.org/fibo/ontology/BP/Process/FinancialContextAndProcess/
https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/AgencyMBSIssuance/
https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/DebtIssuance/
https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/EquitiesIPOIssuance/
https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/IssuanceDocuments/
https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/IssuanceProcess/
https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/MBSIssuance/
https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/MuniIssuance/
https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/PrivateLabelMBSIssuance/
https://spec.edmcouncil.org/fibo/ontology/CAE/CorporateEvents/CorporateActionsEvents/
https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/AssetBaskets/
https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/AssetDerivatives/
https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/BondOptions/
https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/BondReturnSwaps/
https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/EquityForwards/
https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/EquityOptions/
https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/EquitySwaps/
https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/Commodities/
https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommoditiesContracts/
https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommodityForwards/
https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommodityOptions/
https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommoditySpots/
https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommoditySwaps/
https://spec.edmcouncil.org/fibo/ontology/DER/CreditDerivatives/CreditDefaultSwaps/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/ContractsForDifference/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/DerivativesMasterAgreements/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Forwards/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/OptionContractsOnFutures/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Options/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/ParticipationNotes/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/ReturnSwaps/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/RightsAndWarrants/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Spots/
https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Swaptions/
https://spec.edmcouncil.org/fibo/ontology/DER/ExchangeTradedDerivatives/DerivativesStandardizedTerms/
https://spec.edmcouncil.org/fibo/ontology/DER/ExchangeTradedDerivatives/ExchangeTradedOptions/
https://spec.edmcouncil.org/fibo/ontology/DER/ExchangeTradedDerivatives/Futures/
https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxContracts/
https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxForwards/
https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxOptions/
https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxSpots/
https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxSwaps/
https://spec.edmcouncil.org/fibo/ontology/DER/RateDerivatives/ForwardRateAgreements/
https://spec.edmcouncil.org/fibo/ontology/DER/RateDerivatives/InflationSwaps/
https://spec.edmcouncil.org/fibo/ontology/DER/RateDerivatives/IROptions/
https://spec.edmcouncil.org/fibo/ontology/DER/RateDerivatives/OTCIndexOptions/
https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditFacilities/
https://spec.edmcouncil.org/fibo/ontology/FND/AgreementsExt/ContractElements/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/EconomicResources/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/MarketTransactions/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/QuantifiedResources/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/REAClaimsEvents/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/REATransactions/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/SecuritiesTransactions/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/SupplyTransactions/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/TransactionAccounting/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/TransactionEvents/
https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/TransactionsExtended/
https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/Values/
https://spec.edmcouncil.org/fibo/ontology/LOAN/Loans/LoanBasicTerms/
https://spec.edmcouncil.org/fibo/ontology/LOAN/Loans/LoansRegulatory/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/ConstructionLoansTemporal/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoanPhases/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoansEvents/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoansStatus/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoansTemporal/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/CommercialLoans/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/ConstructionLoans/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/CreditProducts/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/GeneralLoans/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/LoanProducts/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/MortgageLoans/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/PersonalLoans/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/SecuredLoans/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/StudentLoans/
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/UnsecuredLoans/
https://spec.edmcouncil.org/fibo/ontology/MD/CIVTemporal/FundsTemporal/
https://spec.edmcouncil.org/fibo/ontology/MD/DebtTemporal/DebtAnalytics/
https://spec.edmcouncil.org/fibo/ontology/MD/DebtTemporal/DebtPricingYields/
https://spec.edmcouncil.org/fibo/ontology/MD/DerivativesTemporal/ETOptionsTemporal/
https://spec.edmcouncil.org/fibo/ontology/MD/DerivativesTemporal/FuturesTemporal/
https://spec.edmcouncil.org/fibo/ontology/MD/EquityTemporal/EquityPricing/
https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/InstrumentTemporalTerms/
https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/SecurityCreditStatuses/
https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/SecurityTradingStatuses/
https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/TemporalConcepts/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/AssetBackedSecurities/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/AssetBackedSPVs/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/CBOs/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/CDOs/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/CLOs/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/CMOs/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/LoanParticipationNotes/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/PoolBackedSecurities/
https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/SyntheticCDOs/
https://spec.edmcouncil.org/fibo/ontology/SEC/Equities/DepositaryReceipts/
```

**Comment by VladimirAlexiev** [comment]
The total dev.ttl.zip has 98 prefixes not defined in fibo-quickstart. I checked, the first URL below is used in quickstart so should be defined.
These are probably the same 97 as listed above plus one extra (haven't checked which one):
```ttl
@prefix fibo-bp-iss-ambs: <https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/AgencyMBSIssuance/> .
@prefix fibo-bp-iss-dbti: <https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/DebtIssuance/> .
@prefix fibo-bp-iss-doc: <https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/IssuanceDocuments/> .
@prefix fibo-bp-iss-ipo: <https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/EquitiesIPOIssuance/> .
@prefix fibo-bp-iss-mbs: <https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/MBSIssuance/> .
@prefix fibo-bp-iss-muni: <https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/MuniIssuance/> .
@prefix fibo-bp-iss-pmbs: <https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/PrivateLabelMBSIssuance/> .
@prefix fibo-bp-iss-prc: <https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/IssuanceProcess/> .
@prefix fibo-bp-prc-fcp: <https://spec.edmcouncil.org/fibo/ontology/BP/Process/FinancialContextAndProcess/> .
@prefix fibo-cae-ce-cae: <https://spec.edmcouncil.org/fibo/ontology/CAE/CorporateEvents/CorporateActionsEvents/> .
@prefix fibo-der-ass-ass: <https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/AssetDerivatives/> .
@prefix fibo-der-ass-bas: <https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/AssetBaskets/> .
@prefix fibo-der-ass-bo: <https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/BondOptions/> .
@prefix fibo-der-ass-brs: <https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/BondReturnSwaps/> .
@prefix fibo-der-ass-eqf: <https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/EquityForwards/> .
@prefix fibo-der-ass-eqo: <https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/EquityOptions/> .
@prefix fibo-der-ass-eqs: <https://spec.edmcouncil.org/fibo/ontology/DER/AssetDerivatives/EquitySwaps/> .
@prefix fibo-der-com-com: <https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/Commodities/> .
@prefix fibo-der-com-ctr: <https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommoditiesContracts/> .
@prefix fibo-der-com-fwd: <https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommodityForwards/> .
@prefix fibo-der-com-opt: <https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommodityOptions/> .
@prefix fibo-der-com-sp: <https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommoditySpots/> .
@prefix fibo-der-com-sw: <https://spec.edmcouncil.org/fibo/ontology/DER/CommoditiesDerivatives/CommoditySwaps/> .
@prefix fibo-der-cr-cds: <https://spec.edmcouncil.org/fibo/ontology/DER/CreditDerivatives/CreditDefaultSwaps/> .
@prefix fibo-der-der-cof: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/OptionContractsOnFutures/> .
@prefix fibo-der-der-fwd: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Forwards/> .
@prefix fibo-der-der-opt: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Options/> .
@prefix fibo-der-der-rsw: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/ReturnSwaps/> .
@prefix fibo-der-der-sp: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Spots/> .
@prefix fibo-der-der-spn: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Swaptions/> .
@prefix fibo-der-drc-cfd: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/ContractsForDifference/> .
@prefix fibo-der-drc-ma: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/DerivativesMasterAgreements/> .
@prefix fibo-der-drc-pn: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/ParticipationNotes/> .
@prefix fibo-der-drc-raw: <https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/RightsAndWarrants/> .
@prefix fibo-der-etd-eto: <https://spec.edmcouncil.org/fibo/ontology/DER/ExchangeTradedDerivatives/ExchangeTradedOptions/> .
@prefix fibo-der-etd-fut: <https://spec.edmcouncil.org/fibo/ontology/DER/ExchangeTradedDerivatives/Futures/> .
@prefix fibo-der-etd-std: <https://spec.edmcouncil.org/fibo/ontology/DER/ExchangeTradedDerivatives/DerivativesStandardizedTerms/> .
@prefix fibo-der-fx-ctr: <https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxContracts/> .
@prefix fibo-der-fx-fwd: <https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxForwards/> .
@prefix fibo-der-fx-opt: <https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxOptions/> .
@prefix fibo-der-fx-sp: <https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxSpots/> .
@prefix fibo-der-fx-sw: <https://spec.edmcouncil.org/fibo/ontology/DER/FxDerivatives/FxSwaps/> .
@prefix fibo-der-rat-fra: <https://spec.edmcouncil.org/fibo/ontology/DER/RateDerivatives/ForwardRateAgreements/> .
@prefix fibo-der-rat-indo: <https://spec.edmcouncil.org/fibo/ontology/DER/RateDerivatives/OTCIndexOptions/> .
@prefix fibo-der-rat-iro: <https://spec.edmcouncil.org/fibo/ontology/DER/RateDerivatives/IROptions/> .
@prefix fibo-der-rtd-isw: <https://spec.edmcouncil.org/fibo/ontology/DER/RateDerivatives/InflationSwaps/> .
@prefix fibo-fbc-dae-crf: <https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditFacilities/> .
@prefix fibo-fnd-agrx-ele: <https://spec.edmcouncil.org/fibo/ontology/FND/AgreementsExt/ContractElements/> .
@prefix fibo-fnd-arr-arr: <https://spec.edmcouncil.org/fibo/ontology/FND/Arrangements/Documents/> .
@prefix fibo-fnd-txn-acc: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/TransactionAccounting/> .
@prefix fibo-fnd-txn-cle: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/REAClaimsEvents/> .
@prefix fibo-fnd-txn-ecr: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/EconomicResources/> .
@prefix fibo-fnd-txn-ev: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/TransactionEvents/> .
@prefix fibo-fnd-txn-mkt: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/MarketTransactions/> .
@prefix fibo-fnd-txn-qr: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/QuantifiedResources/> .
@prefix fibo-fnd-txn-rea: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/REATransactions/> .
@prefix fibo-fnd-txn-sec: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/SecuritiesTransactions/> .
@prefix fibo-fnd-txn-sup: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/SupplyTransactions/> .
@prefix fibo-fnd-txn-txnx: <https://spec.edmcouncil.org/fibo/ontology/FND/TransactionsExt/TransactionsExtended/> .
@prefix fibo-fnd-utl-val: <https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/Values/> .
@prefix fibo-loan-loan-loan: <https://spec.edmcouncil.org/fibo/ontology/LOAN/Loans/LoanBasicTerms/> .
@prefix fibo-loan-loan-reg: <https://spec.edmcouncil.org/fibo/ontology/LOAN/Loans/LoansRegulatory/> .
@prefix fibo-loan-tem-cl: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/ConstructionLoansTemporal/> .
@prefix fibo-loan-tem-ev: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoansEvents/> .
@prefix fibo-loan-tem-ph: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoanPhases/> .
@prefix fibo-loan-tem-sta: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoansStatus/> .
@prefix fibo-loan-tem-tem: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoansTemporal/> .
@prefix fibo-loan-typ-cl: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/ConstructionLoans/> .
@prefix fibo-loan-typ-com: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/CommercialLoans/> .
@prefix fibo-loan-typ-cr: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/CreditProducts/> .
@prefix fibo-loan-typ-gen: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/GeneralLoans/> .
@prefix fibo-loan-typ-mtg: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/MortgageLoans/> .
@prefix fibo-loan-typ-per: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/PersonalLoans/> .
@prefix fibo-loan-typ-prod: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/LoanProducts/> .
@prefix fibo-loan-typ-sec: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/SecuredLoans/> .
@prefix fibo-loan-typ-stu: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/StudentLoans/> .
@prefix fibo-loan-typ-uns: <https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/UnsecuredLoans/> .
@prefix fibo-md-civx-fun: <https://spec.edmcouncil.org/fibo/ontology/MD/CIVTemporal/FundsTemporal/> .
@prefix fibo-md-dbtx-aly: <https://spec.edmcouncil.org/fibo/ontology/MD/DebtTemporal/DebtAnalytics/> .
@prefix fibo-md-dbtx-py: <https://spec.edmcouncil.org/fibo/ontology/MD/DebtTemporal/DebtPricingYields/> .
@prefix fibo-md-der-fut: <https://spec.edmcouncil.org/fibo/ontology/MD/DerivativesTemporal/FuturesTemporal/> .
@prefix fibo-md-derx-eto: <https://spec.edmcouncil.org/fibo/ontology/MD/DerivativesTemporal/ETOptionsTemporal/> .
@prefix fibo-md-eqx-pri: <https://spec.edmcouncil.org/fibo/ontology/MD/EquityTemporal/EquityPricing/> .
@prefix fibo-md-temx-crs: <https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/SecurityCreditStatuses/> .
@prefix fibo-md-temx-ins: <https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/InstrumentTemporalTerms/> .
@prefix fibo-md-temx-tem: <https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/TemporalConcepts/> .
@prefix fibo-md-temx-trs: <https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/SecurityTradingStatuses/> .
@prefix fibo-sec-dbt-ab-abs: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/AssetBackedSecurities/> .
@prefix fibo-sec-dbt-ab-cbo: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/CBOs/> .
@prefix fibo-sec-dbt-ab-cdo: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/CDOs/> .
@prefix fibo-sec-dbt-ab-clo: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/CLOs/> .
@prefix fibo-sec-dbt-ab-cmo: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/CMOs/> .
@prefix fibo-sec-dbt-ab-lpn: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/LoanParticipationNotes/> .
@prefix fibo-sec-dbt-ab-mbs: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities/> .
@prefix fibo-sec-dbt-ab-pbs: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/PoolBackedSecurities/> .
@prefix fibo-sec-dbt-ab-spv: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/AssetBackedSPVs/> .
@prefix fibo-sec-dbt-ab-syn: <https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/SyntheticCDOs/> .
@prefix fibo-sec-eq-dr: <https://spec.edmcouncil.org/fibo/ontology/SEC/Equities/DepositaryReceipts/> .
```

**Comment by dallemang** [comment]
I noticed this today as well, and was about to create an issue about it, but I saw this one, so I'll use it. 

The issue here isn't with quickstart itself, that's a symptom.  The issue is that many files (and I just wrote a script to find out which ones they are, and got the same number as you) don't specify the sm:fileAbbreviation. 

I updated my script to actually make the changes to add that in.   I want to do this before the Q2 release goes out.  I'm going to assign this one to me.
